### PR TITLE
[Do not merge] Issue #235 Removing mentions of jimmidyson

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
Steps done for testing the PR

```
```
$ rm -rf vendor
$ make vendor
$ make
$ make test
$ minishift start
$ minishift delete
```

**Comment by budhrg** [comment]
Working fine for me.
Tested as:
```
$ make clean
$ make 
$ make test
$ make start
$ # deploy app
$ make stop
$ make start
$ # check app
```

LGTM :+1:

File: glide.lock [hunk 1]
```diff
@@ -341,7 +341,7 @@ imports:
   version: 1560c1005499d61b80f865c04d39ca7505bf7f0b
 - name: github.com/spf13/viper
   version: c975dc1b4eacf4ec7fdbf0873638de5d090ba323
-  repo: https://github.com/jimmidyson/viper.git
+  repo: https://github.com/spf13/viper.git
```
Comment (hferentschik) [inline glide.lock]: this does not look right. Did you remove the lock file and regenerate it after updating _glide.yaml_ or did you just do a find and replace.

File: glide.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ import:
 - package: github.com/olekukonko/tablewriter
 - package: github.com/pkg/browser
 - package: github.com/spf13/viper
-  repo: https://github.com/jimmidyson/viper.git
+  repo: https://github.com/spf13/viper.git
```
Comment (hferentschik) [inline glide.yaml]: If we don't use a custom repo we don't need the explicit _repo_ part. But we might need to set a version. See also https://github.com/minishift/minishift/blob/master/docs/developing.md#updating-dependencies

File: glide.lock (hunk 1 above)
Comment (hferentschik) [inline glide.lock]: I am also suspicious since we are referencing a commit sha here, but this sha might not be in the main repo.

Comment (budhrg) [inline glide.lock]: @hferentschik Oh ya, I tested with removing glide.lock. but found that `spf13/viper.git` version is same as ` c975dc1b4eacf4ec7fdbf0873638de5d090ba323`.

**Comment by hferentschik** [comment]
@budhrg One cannot just "change" _glide.lock_, you want to make the change to _glide.yaml_ and then re-generate the lock file.

**Comment by budhrg** [comment]
@hferentschik , I did remove `glide.lock` file and regenerated with the lala's `glide.yaml` changes. It looks fine.

File: glide.yaml (hunk 2 above)
Comment (budhrg) [inline glide.yaml]: > If we don't use a custom repo we don't need the explicit repo part. 

Yes

Setting version will help to get to stable point.

**Comment by hferentschik** [comment]
So if you actually follow the proper procedure to do switch to the official Viper repo, you get:

    CGO_ENABLED=0 GOARCH=amd64 GOOS=darwin go build --installsuffix cgo -ldflags="-X github.com/minishift/minishift/    pkg/version.version=1.0.0-beta.2 -X github.com/minishift/minishift/pkg/version.openshiftVersion=v1.3.1 -s -w     -extldflags '-static'" -o ./out/darwin-amd64/minishift ./cmd/minikube
    # github.com/minishift/minishift/vendor/github.com/spf13/viper
    vendor/github.com/spf13/viper/viper.go:741: undefined: mapstructure.StringToTimeDurationHookFunc
    make: *** [out/darwin-amd64/minishift] Error 2

So something is still astray. If we want to remove the usage of jimmy's repo, we need to dig a bit deeper here and see whether we can resolve this in any other way (understand what jimmy's repo actually changes and whether there some alternative in the official Viper repo or whether we can inline a helper or so)

**Comment by LalatenduMohanty** [comment]
@hferentschik Right, I had manually modified glide.lock , now as per your comment I could reproduce the error.

**Comment by LalatenduMohanty** [comment]
Closing the PR now. Will resend after fixing the issue.

**Pull request closed without merging**